src/conditionals/programs_with_decision_points/medium/q8/MainTest.java

```diff
@@ -0,0 +1,70 @@
+package conditionals.programs_with_decision_points.medium.q8;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+public class MainTest extends BaseTest {
+  // Parsons
+  public Clause[] testSentence() {
+    TestOption.isInputTest = true;
+    TestOption.defaultInput = "1";
+    return new Clause[] {
+        new StringLiteral("Which place did you finish\\? \\(Ex. 1, 2, 3, etc\\)\\:"),
+        new NewLine(),
+        new PlaceHolder(),
+        new PlaceHolder(),
```

Comment:
See previous, you should not need this PlaceHolder and corresponding NewLine() below

Reply:
https://github.com/canvas-gamification/canvas-gamification-junit-tests/pull/258